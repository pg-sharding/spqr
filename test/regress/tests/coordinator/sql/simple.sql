-- UNREGISTER ROUTER ALL
REGISTER ROUTER r1 ADDRESS regress_router_1:7000;