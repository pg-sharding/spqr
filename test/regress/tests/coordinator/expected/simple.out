-- UNREGISTER ROUTER ALL
REGISTER ROUTER r1 ADDRESS regress_router_1:7000;
              register routers              
--------------------------------------------
 router r1-regress_router_1:7000 registered
(1 row)

